packages/ckeditor5-dev-build-tools/src/plugins/splitCss.ts

```diff
@@ -30,10 +30,35 @@ export interface RollupSplitCssOptions {
  */
 const filter = createFilter( [ '**/*.css' ] );
 
-/**
- * Regular expression to match CSS variables.
- */
-const VARIABLE_DEFINITION_REGEXP = /--([\w-]+)/gm;
+const commonPurgeCssOptions = {
+	fontFace: true,
+	keyframes: true,
+	variables: true
+};
+
+const purgeCssOptionsForContent = {
+	...commonPurgeCssOptions,
+	content: [],
+	safelist: { deep: [ /^ck-content/ ] },
+	blocklist: []
+};
+
+const purgeCssOptionsForEditor = {
+	...commonPurgeCssOptions,
+	// Pseudo class`:where` is preserved only if the appropriate html structure matches the CSS selector.
+	// It's a temporary solution to avoid removing selectors for Show blocks styles where `:where` occurs.
+	// See: https://github.com/FullHuman/purgecss/issues/978
+	content: [ {
+		raw: '<html><body><div class="ck ck-editor__editable ck-editor__editable_inline ck-show-blocks">' +
```

Comment:
Would PurgeCSS handle this HTML if we used [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) and format this HTML nicely?

Reply:
Yes, it works with template literals. 96d3bb7